src/deprecation.js

```diff
@@ -0,0 +1,37 @@
+var core = require('./core'),
+    math = core.math,
+    // plugins:
+    extras         = require('./extras'),
+    filters        = require('./filters'),
+    interaction    = require('./interaction'),
+    loaders        = require('./loaders'),
+    spine          = require('./spine'),
+    text           = require('./text');
+
+core.Stage = function ()
+{
+        window.console.warn("You don't need to use a PIXI Stage any more, you can simply render any container");
+        return new core.Container();
+};
+
+core.SpriteBatch = function ()
+{
+        window.console.warn("SpriteBatch doesn't exist any more, please use the new ParticleContainer instead");
```

Comment:
Why do these warn, but not perform the previous action like Stage does? This needs to also have `return new core.ParticleContainer();`